_episodes/02-numpy.md

```diff
@@ -665,38 +665,38 @@ which is the average inflammation per patient across all days.
 > with NumPy.
 >
 > The `numpy.diff()` function takes an array and returns the differences
-> between two successive values. First we consider a one-dimensional
-> array of length 5. This could be part of some row `i` of our inflammation data,
-> i.e. `row_start = data[i,:5]`.
->
+> between two successive values. Let's use it to examine the changes
+> each day across the first week of patient 3 from our inflammation dataset.
+> 
 > ~~~
-> row_start = numpy.array([ 0,  2,  5,  9, 14])
+> patient3_week1 = data[3, :7] 
```

Comment:
Looking at this in the rendered lesson now, I am thinking we should add a `print(patient3_week1)` to this Python block and add an output block:
```
>
> ~~~
>  [0. 0. 2. 0. 4. 2. 2.]
> ~~~
> {: .output}
```



This way, people can see the connection between the numbers in the array and the calculations shown below.